core/nginx/conf/nginx.conf

```diff
@@ -252,7 +252,7 @@ http {
       {% endif %}
       {% endif %}
 
-      {% if API %}
+      {% if API == 'true' %}
```

Comment:
This works, but I think using socrate's output in `/config.py` would be better.
Instead of 
```
system.set_env()
args = os.environ.copy()
```
this should do the trick (API would the be normalized to boolean):
```
args = system.set_env()
```

Reply:
Ah. Yes. I also prefer this. It might be worthwhile to check if we also use this in the other images.